lib/sqlalchemy/dialects/oracle/base.py

```diff
@@ -2051,8 +2051,16 @@ def _table_options_query(
     ):
         query = select(
             dictionary.all_tables.c.table_name,
-            dictionary.all_tables.c.compression,
-            dictionary.all_tables.c.compress_for,
+            (
+                dictionary.all_tables.c.compression
+                if self._supports_table_compression
+                else sql.literal_column("NULL").label("compression")
```

Comment:
You can use sql.null() here

Reply:
It's my first week with sqlalchemy, thanks for the tip 👍